<ui version="4.0" >
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Configure</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QListWidget" name="ui_pageList" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="currentRow" >
        <number>-1</number>
       </property>
       <item>
        <property name="text" >
         <string>General</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Theme</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Engines</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="ui_stackedWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <widget class="QWidget" name="ui_generalPage" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>340</width>
          <height>340</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="ui_themePage" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>340</width>
          <height>340</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="ui_enginePage" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>340</width>
          <height>340</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="ui_engineBox" >
           <property name="title" >
            <string>Chess Engines</string>
           </property>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QListWidget" name="ui_engineList" />
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QPushButton" name="ui_addEngine" >
                <property name="text" >
                 <string>Add...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ui_modifyEngine" >
                <property name="text" >
                 <string>Modify...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ui_deleteEngine" >
                <property name="text" >
                 <string>Delete...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ui_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ui_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
